.gitignore

```diff
@@ -5,7 +5,7 @@ dependency-reduced-pom.xml
 .idea/
 *.iml
 *.~*
-
+*~
```

Comment:
General note: I usually recommend to put editor-specific ignore patterns into $HOME/.gitignore because then they work for all projects and you don't have to rely on the maintainer to accept your favourite pattern.

Reply:
Thanks, didn't know this!